main.tex

```diff
@@ -18,6 +18,49 @@
 \begin{document}
 
 \maketitle
+% 2.3 in der Gliederung
+\subsection{Start in ausgewählten Regionen}
 
+Im Anschluss an den formalen Start fingen erneute Debatten um "Net-Neutrality" an Beoden zu gewinnen. Facebooks Dienst sei nicht konform mit dem Bestreben eines freien Internets, trotz der Bewerbung des Produkts als "kostenloses Facebook"\cite[3]{prasad2017}
+ , später als ein "step towards digital equality".
+
+\medskip
+
+In der ländlichen Bevölkerung gab es auch geteilte Meinungen hinsichtlich des Zugangs zum Internet, zumindest bei jenen, die sich selbst informiert haben oder (von zum Beispiel Nachrichtendiensten wie "Digit.in") \textcite{digitYT}
+informiert werden. Hier kommt es jdeoch zumeist zur Ignoranz des großen Ganzen, der Dienst wird als positiv entgegengenommen und genutzt, Aktionen wie die wortwörtlich vorgeschriebene Unterschreibung zugunsten von Facebook in der Debatte um Netzneutralität \textcite{ndtvYT}
```

Comment:
```suggestion
informiert werden. Hier kommt es jedoch zumeist zur Ignoranz des großen Ganzen, der Dienst wird als positiv entgegengenommen und genutzt, Aktionen, wie die wortwörtlich vorgeschriebene Unterschreibung zugunsten von Facebook in der Debatte um Netzneutralität, \textcite{ndtvYT}
```

Reply:
Ist ein ganz schön langer Satz, könnte man der Übersichtlichkeit halber evtl in 2 Sätze fassen.